docs/operators/fixedWidthParserOperator.md

```diff
@@ -0,0 +1,170 @@
+            "length": "3",
+            "padding": "0"
+        },
+        {
+            "name": "campaignId",
+            "type": "Integer",
+            "length": "3",
+            "padding": " "
+
+        },
+        {
+            "name": "bidPrice",
+            "type": "Double",
+            "length": "3"
+        },
+        {
+            "name": "startDate",
+            "type": "Date",
+            "format": "yyyy-MM-dd HH:mm:ss",
+            "length": "19"
+
+        },
+        {
+            "name": "endDate",
+            "type": "Date",
+            "format": "dd/MM/yyyy",
+            "length": "10"
+        },
+        {
+            "name": "securityCode",
+            "type": "Long",
+            "length": "5"
+        },
+        {
+            "name": "active",
+            "type": "Boolean",
+            "length": "1",
+            "trueValue": "y",
+            "falseValue": "n"
+        },
+        {
+            "name": "optimized",
+            "type": "Boolean",
+            "length": "1",
+            "trueValue": "y",
+            "falseValue": "n"
+        },
+        {
+            "name": "valid",
+            "type": "String",
+            "length": "10",
+            "padding":" "
+        }
+    ]
+}
+
+
+```
+
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *jsonSchema* | [Schema](https://github.com/apache/apex-malhar/blob/master/contrib/src/main/java/com/datatorrent/contrib/parser/FixedWidthSchema.java)  describing fixed width data. Based on schema definition the operator will parse the incoming record to object map and POJO.
+Valid records will be emitted as POJO / map while invalid ones are emitted on error port with error message. | String | YES | N/A |
+
+## Platform Attributes that influences operator behavior
+
+| **Attribute** | **Description** | **Type** | **Mandatory** |
+| -------- | ----------- | ---- | ------------------ |
+| *out.TUPLE_CLASS* | TUPLE_CLASS attribute on output port which tells operator the class of POJO which need to be emitted | Class| Yes |
+
+## <a name="dataTypes"></a>Supported DataTypes in Schema
+  * Integer
+  * Long
+  * Double
+  * Character
+  * String
+  * Boolean
+  * Date
+  * Float
+
+
+## Ports
+
+| **Port** | **Description** | **Type** | **Mandatory** |
+| -------- | ----------- | ---- | ------------------ |
+| *in* | Tuples that needs to be parsed are recieved on this port | byte[] | Yes |
+| *out* | Valid Tuples that are emitted as pojo | Object (POJO) | No |
+| *parsedOutput* | Valid Tuples that are emitted as map | Map | No |
+| *err* | Invalid Tuples are emitted with error message | KeyValPair <String, String\> | No |
+
+## Partitioning
+Fixed Width Parser is both statically and dynamically partitionable.
+### Static Partitioning
+This can be achieved in 2 ways
+
+1. Specifying the partitioner and number of partitions in the populateDAG() method
```

Comment:
highlight literals like populateDAG() in some way; use the same convention consistently throughout